instructure/instructure-ui
# [BREAKING] remove renderName and nameBackground prop from TopNavBar.Brand

INSTUI-4017

BREAKING CHANGE:
`renderName` and `nameBackground` prop is not available on TopNavBar.Brand anyomore

should only be merged right before the v9 release

test plan:

- check docs for TopNavBar: any mention of renderName or nameBackground for TopNavBar.Brand should be removed
- add `renderName` to `<TopNavBar.Brand />`: there should be a console error/warning about the deprecated prop with a url to the upgrade guide
- add `nameBackground` to `<TopNavBar.Brand />`: there should be a console error/warning about the deprecated prop with a url to the upgrade guide
- check the upgrade guide
